src/test/scala/dsptools/numbers/FixedPrecisionChangerSpec.scala

```diff
@@ -45,34 +45,34 @@ class RemoveMantissaTester(c: RemoveMantissa, inValue: Double, outValue: Double)
 class FixedPrecisionChangerSpec extends FreeSpec with Matchers {
   "assignment of numbers with differing binary points seems to work as I would expect" - {
     "here we assign to a F8.1 from a F8.3" in {
-      chisel3.iotesters.Driver(() => new FixedPrecisionChanger(8, 3, 8, 1)) { c=>
+      dsptools.Driver.execute(() => new FixedPrecisionChanger(8, 3, 8, 1), Array.empty[String]) { c =>
```

Comment:
Also, will these tests not work if you use the iotesters.Driver? What makes dsptools Driver different?